app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/SpotType.java

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.sweng.eventmanager.repository.data;
+
+import java.security.cert.CertPathValidatorException;
+
+/**
+ * Represents all type of spot
+ *
+ * @author Robin Zbinden (274236)
+ * @author Stanislas Jouven (260580)
+ */
+public enum SpotType {
```

Comment:
Maybe we will want to leave this as a string so that the event organizer can create a spot of any type, event the ones we didn't think about (but for now an enum is fine)